hoomd/md/NeighborList.cc

```diff
@@ -1742,6 +1764,253 @@ void NeighborList::updateMemoryMapping()
     }
 #endif
 
+pybind11::array_t<uint32_t> NeighborList::getLocalPairListPython(uint64_t timestep)
+    {
+    compute(timestep);
+
+    bool third_law = getStorageMode() == NeighborList::half;
+
+    ArrayHandle<unsigned int> h_n_neigh(getNNeighArray(), access_location::host, access_mode::read);
+    ArrayHandle<unsigned int> h_nlist(getNListArray(), access_location::host, access_mode::read);
+    ArrayHandle<size_t> h_head_list(getHeadList(), access_location::host, access_mode::read);
+
+    auto* pair_list = new std::vector<vec2<uint32_t>>();
+
+    pybind11::capsule delete_when_done(pair_list,
+                                       [](void* f)
+                                       {
+                                           auto data
+                                               = reinterpret_cast<std::vector<vec2<uint32_t>>*>(f);
+                                           delete data;
+                                       });
+
+    for (unsigned int i = 0; i < m_pdata->getN(); i++)
+        {
+        const size_t myHead = h_head_list.data[i];
+        const unsigned int size = h_n_neigh.data[i];
+        for (unsigned int k = 0; k < size; k++)
+            {
+            const unsigned int j = h_nlist.data[myHead + k];
```

Comment:
```suggestion
        const size_t my_head = h_head_list.data[i];
        const unsigned int size = h_n_neigh.data[i];
        for (unsigned int k = 0; k < size; k++)
            {
            const unsigned int j = h_nlist.data[my_head + k];
```
Use lowercase and underscores for variable names.